fluidkeys/secretreceive.go

```diff
@@ -51,6 +51,11 @@ func secretReceive() exitCode {
 	var sawError bool = false
 
 	for _, key := range keys {
+		if !Config.ShouldPublishToAPI(key.Fingerprint()) {
+			message := "Not published in Fluidkeys directory"
+			out.Print("📪 " + displayName(&key) + ": " + colour.Warning(message) + "\n")
```

Comment:
Suggest that we don't use the mailbox emoji... perhaps rather ⛔ ?

Reply:
nice yeah